src/components/Contacts/ContactRow.stories.tsx

```diff
@@ -18,7 +18,7 @@ export default {
 export const Default: Story = () => {
   const contact = gqlMock<ContactRowFragment>(ContactRowFragmentDoc);
 
-  return <ContactRow contact={contact} />;
+  return <ContactRow contact={contact} onContactSelected={() => {}} />;
```

Comment:
This could be wired up to a story action. Not important though...

Reply:
Yeah maybe, although the ContactRow doesn't change at all when the row is selected, so I don't think wiring this to a story action would change how this appears in Storybook.

Reply:
I think the story action is less focused on how it appears in the storybook and more so on if logic is called when expected. Wiring it up to the action could test that the callback function is fired when expected inside the component.

Reply:
But like Scotty mentioned, not that important